### PR TITLE
fix(hds compliance): add a FAQ entry about 'point de contact'

### PR DESCRIPTION
**Pull request by Frzk**
First proposal to address the "point de contact" questions that keep coming.
Please let me know if it seems ok for you, if something needs to be updated, or if something is missing :)

### CODE REVIEW

File: src/_posts/compliance/2000-01-01-hds.md [hunk 1]
```diff
@@ -70,3 +76,9 @@ La certification HDS de Scalingo couvre les périmètres suivants :
 Veuillez toujours vous référer à la page actuelle pour obtenir les dernières informations sur notre conformité HDS
 
 [Télécharger le certificat HDS 1.1 en vigueur](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
+
+## 🇫🇷 Qu'est-ce qu'un "Point de contact professionnel de santé" ? {#fr-hds-point-of-contact-health-professional}
+
+La certification HDS impose à Scalingo de maintenir une liste de *points de contact professionnel de santé* pour chacun de ses clients.
+Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer des relations avec le patient, etc.)
```
Comment (yanjost) [inline src/_posts/compliance/2000-01-01-hds.md:83]:
```suggestion
Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer un incident de sécurité, etc.)
```

File: src/_posts/compliance/2000-01-01-hds.md [hunk 2]
```diff
@@ -70,3 +76,9 @@ La certification HDS de Scalingo couvre les périmètres suivants :
...
+En outre, Scalingo doit être en capacité de transmettre sans délai cette liste à l’autorité compétente sur demande, notamment en cas de suspension ou de retrait de la certification HDS.
```
Comment (yanjost) [inline src/_posts/compliance/2000-01-01-hds.md:84]: Source ?

File: src/_posts/compliance/2000-01-01-hds.md [hunk 3]
```diff
@@ -58,7 +64,7 @@ Le numéro de certificat est 38436, il a été délivré par [LNE](https://www.l
 
 La certification HDS de Scalingo couvre les périmètres suivants :
 
-1. Mise à disposition et maintien en condition opérationnelle des sites physiques permettant d’héberger l’infrastructure matérielle du système d’information utilisé pour le traitement des données de santé ;
+1. Mise à disposition et maintien en condition opératThe search found this: ionnelle des sites physiques permettant d’héberger l’infrastructure matérielle du système d’information utilisé pour le traitement des données de santé ;
```
Comment (yanjost) [inline src/_posts/compliance/2000-01-01-hds.md:67]: ;)

File: src/_posts/compliance/2000-01-01-hds.md [hunk 4]
```diff
@@ -38,6 +38,12 @@ Please always refer the current page to get the latest compliance information
 
 [Download the current HDS 1.1 certificate](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
 
+# 🇬🇧 What is a "Point of Contact health Professional" ("Point de contact professionnel de santé" in French)? {#en-hds-point-of-contact-health-professional}
+
+The HDS certification enjoins Scalingo to maintain a list of *Points of contact health professional* for each of its customer.
```
Comment (yanjost) [inline src/_posts/compliance/2000-01-01-hds.md:43]:
```suggestion
The HDS certification enjoins Scalingo to have a up-to-date *Health Professional Point of Contact* for each of its customers.
```

File: src/_posts/compliance/2000-01-01-hds.md [hunk 5]
```diff
@@ -38,6 +38,12 @@ Please always refer the current page to get the latest compliance information
...
+These points of contact must be able to designate an health professional to Scalingo when required (for example, to access to health data or to manage the relationship with the client, ...)
```
Comment (yanjost) [inline src/_posts/compliance/2000-01-01-hds.md:44]:
```suggestion
This Point of Contact is a person able to designate a Health Professional to Scalingo when the situation requires it (for example, to access to Health Data or to manage a Security Incident, ...)
```

**Review by yanjost — CHANGES_REQUESTED** [review]
I reworded some sentences, tell me what you think

I think there is a "last updated date" field in the yaml frontmatter, you should update it

File: src/_posts/compliance/2000-01-01-hds.md (hunk 2 above)
Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md:84]: Page 17 of this pdf file, which seems legit: https://esante.gouv.fr/sites/default/files/media_entity/documents/hds_referentiel_de_certification_v1.1f_mai2018.pdf

Please tell me if I should use another source :)

File: src/_posts/compliance/2000-01-01-hds.md (hunk 1 above)
Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md]: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

File: src/_posts/compliance/2000-01-01-hds.md (hunk 4 above)
Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md]: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

File: src/_posts/compliance/2000-01-01-hds.md (hunk 5 above)
Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md]: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

File: src/_posts/compliance/2000-01-01-hds.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md]: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

Sorry for this :(

File: src/_posts/compliance/2000-01-01-hds.md (hunk 2 above)
Comment (yanjost) [reply src/_posts/compliance/2000-01-01-hds.md:84]: You couldn't find better source in fact :)
I suggest you mention it in the french and english versions (there is an english translation of the referential too)

Reference it like this

> En outre, Scalingo doit être en capacité de transmettre sans délai cette liste à l’autorité compétente sur demande, notamment en cas de suspension ou de retrait de la certification HDS (Référentiel HDS 1.1.1f, exigence 4.5.4).

Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md:84]: Should I add a link the referential? (if so, could you please provide them?)

File: src/_posts/compliance/2000-01-01-hds.md [hunk 6]
```diff
@@ -38,6 +38,12 @@ Please always refer the current page to get the latest compliance information
 
 [Download the current HDS 1.1 certificate](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
 
+# 🇬🇧 What is a "Point of Contact health Professional" ("Point de contact professionnel de santé" in French)? {#en-hds-point-of-contact-health-professional}
```
Comment (yannski) [inline src/_posts/compliance/2000-01-01-hds.md:41]:
```suggestion
# 🇬🇧 What is a "Health Professional Point of Contact" ("Point de contact professionnel de santé" in French)? {#en-hds-point-of-contact-health-professional}
```

File: src/_posts/compliance/2000-01-01-hds.md [hunk 7]
```diff
@@ -38,6 +38,12 @@ Please always refer the current page to get the latest compliance information
...
+
+The HDS certification enjoins Scalingo to have a up-to-date *Health Professional Point of Contact* for each of its customers.
+This Point of Contact is a person able to designate a Health Professional to Scalingo when the situation requires it (for example, to access to Health Data or to manage a Security Incident, ...)
```
Comment (yannski) [inline src/_posts/compliance/2000-01-01-hds.md:44]:
```suggestion
This Point of Contact is a person able to designate a Health Professional to Scalingo when the situation requires it (for example, to access Health Data or to manage a Security Incident, ...)
```

File: src/_posts/compliance/2000-01-01-hds.md (hunk 2 above)
Comment (yannski) [reply src/_posts/compliance/2000-01-01-hds.md:84]: @Frzk IMHO linking the referential is not necessary. But you should mention the referential article number like Yannick did ie "Référentiel HDS 1.1.1f, exigence 4.5.4"

Comment (Frzk) [reply src/_posts/compliance/2000-01-01-hds.md]: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/d5252491897da1d156b7e0e5ea7e81b8e7027781

**Review by yannski — APPROVED** [review]

**Pull request merged**